cache/remotecache/gha/gha.go

```diff
@@ -368,6 +368,15 @@ func (p *ciProvider) ReaderAt(ctx context.Context, desc ocispecs.Descriptor) (co
 	return &readerAt{ReaderAtCloser: rac, desc: desc}, nil
 }
 
+func (p *ciProvider) Info(ctx context.Context, dgst digest.Digest) (content.Info, error) {
+	_, err := p.loadEntry(ctx, ocispecs.Descriptor{Digest: dgst})
+	if err != nil {
+		return content.Info{}, err
+	}
+	// TODO: Fill Size?
+	return content.Info{Digest: dgst}, nil
```

Comment:
Only `Digest` here.

Reply:
This can be retrieved from `p.desc` . Looks like we could replace the `CheckDescriptor()` method with this new `Info()` method now, but that can remain in follow-up.

Reply:
Oh right, I somehow confused this with `util/contentutil/fetcher.go` and didn't use it because I thought this one will have `Size: 0` 🤦🏻